docs/code/getting-started.md

```diff
@@ -584,11 +600,11 @@ Let’s create a Window with a vertical Grid that contains a Button and a Label:
     grid.add (button);
     grid.add (label);
 
-    window.add (grid);
+    main_window.add (grid);
```

Comment:
Center the grid?

Reply:
Outside the scope of this branch. I haven't touched the grid here